client/ayon_usd/addon.py

```diff
@@ -34,23 +38,48 @@ def tray_start(self):
         Download USD if needed.
         """
         super(USDAddon, self).tray_start()
-        download_usd = is_usd_download_needed()
-        if not download_usd:
-            print(f"get_downloaded_usd_root: {get_downloaded_usd_root()}")
+
+        if not utils.is_usd_download_needed():
+            print(f"usd is allready downloaded")
             return
 
-        from .download_ui import show_download_window
+        # this exists because for some reason there is a downloades.zip where the downloads folder should be. i don't know why it is there but leaving it means we cant download into the location
```

Comment:
Please mark this as a `# TODO`  or `# QUESTION`  or `# NOTE`. Doesn't seem like something we want to keep like this over time.

Also, check PEP8 line lengths with your linter.
```suggestion
        # For some reason there is a downloads.zip where the downloads folder should be. i don't know why it is there but leaving it means we cant download into the location
```